### DISCUSSION TITLE
Aggregation [Aug 15]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#6210] [Discord Threads]
> 🙋‍♂️ **_gdelgado**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### Use rollup; it supports different functions, average being one of them  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1129822588030427197)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
